functions/quality_assessment/quality_assessment.py

```diff
@@ -161,41 +197,26 @@ def get_prompt_list() -> list[dict]:
     for item in prompt_list:
         new_item = {"label": item["prompt-id"]["S"], "value": item["prompt"]["S"]}
         prompt_output_list.append(new_item)
-    return prompt_output_list
+    return {"prompts": prompt_output_list}
```

Comment:
Perhaps this could also be described as a dictionary comprehension

`{"label": item["prompt-id"]["S"], "value": item["prompt"]["S"]} for item in prompt_list}`